sale_default_terms_conditions/tests/test_sale_order.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+
+@tagged("post_install", "-at_install")
+class TestSaleOrder(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestSaleOrder, cls).setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.partner_1 = cls.env["res.partner"].create(
+            {"name": "partner_1", "company_id": cls.env.company.id}
+        )
+        cls.partner_2 = cls.env["res.partner"].create(
+            {"name": "partner_2", "company_id": cls.env.company.id}
+        )
+        cls.partner_1.write(
+            {"sale_order_note": "Sale Default Terms and Conditions Partner"}
+        )
+
+    def test_onchange_partner_id(self):
+
+        self.env["ir.config_parameter"].sudo().set_param(
+            "account.use_invoice_terms", True
+        )
+
+        self.env.company.terms_type = "plain"
+        self.env.company.invoice_terms = "Company Terms"
```

Comment:
Same here. If we install your module with only the required dependency, this line fail.